index.html

```diff
@@ -66,8 +66,210 @@ <h1 id="main-header">I’m Alexandra.
+                </div>
+                <div class="view-project-button">
+                    <button>See Project</button>
+                </div>
+            </section>
+        </section>
+
+        <section class="project-section">
+            <figure>
+                <img src="./Images/gym_fit.png" alt="" id="snap-shot-image">
+            </figure>
+            <section class = "project-showing">
+                <h2>Tonic</h2>
+                <ul class="experience">
+                    <li id="canopy-text">CANOPY</li>
+                    <img src="./Images/Counter.png" alt="" class="counter-img">
+                    <li>Back End Dev</li>
+                    <img src="./Images/Counter.png" alt="" class="counter-img">
+                    <li>2015</li>
+                </ul>
+                <p class="info-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="skill-buttons">
+                    <button>HTML</button>
+                    <button>CSS</button>
+                    <button>JavaScript</button>
+                </div>
+                <div class="view-project-button">
+                    <button>See Project</button>
+                </div>
+            </section>
+        </section>
+
+        <section class="project-section">
+            <figure>
+                <img src="./Images/Snapshoot Portfolio (2).png" alt="" id="snap-shot-image">
+            </figure>
+            <section class="project-showing">
+                <h2>Multi-Post
+                    Stories</h2>
+                <ul class="experience">
+                    <li id="canopy-text">CANOPY</li>
+                    <img src="./Images/Counter.png" alt="" class="counter-img">
+                    <li>Back End Dev</li>
+                    <img src="./Images/Counter.png" alt="" class="counter-img">
+                    <li>2015</li>
+                </ul>
+                <p class="info-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="skill-buttons">
+                    <button>HTML</button>
+                    <button>CSS</button>
+                    <button>JavaScript</button>
+                </div>
+                <div class="view-project-button">
+                    <button>See Project</button>
+                </div>
+            </section>
+        </section>
+    </section>
+
+
+        <!-- ABOUT MYSELF SECTION START  -->
+
+        <section id="about-myself-section">
+            <h2>About Myself</h2>
+            <p class="info-text" id="about-text">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="link" id="about-myself-link">
+                LET'S CONNECT
+            </div>
```

Comment:
- [ ] Please use `hx` tags (i.e. `h2`, `h3`, `h4`, `h5` , `h6`) instead of `div` tag. The purpose of `divs` is to allow you to group elements together. Unless you need to include the div for some structural/organizational reason you're likely better off just using  `hx` tags. 

_Note: keep in mind the hierarchy of these tags e.g.` h3` can't come before` h2 `and `h3 ` can't be placed after` h4` etc._